sysdig/sysdig_clients.go

```diff
@@ -173,6 +184,20 @@ func getIBMSecureVariables(data *schema.ResourceData) (*ibmVariables, error) {
 	return getIBMVariables("secure", data)
 }
 
+func (c *sysdigClients) Configure(ctx context.Context, d *schema.ResourceData) {
+	c.ctx = ctx
+	c.d = d
+}
+
+func (c *sysdigClients) Close() error {
+	ctx, _ := context.WithDeadline(context.Background(), time.Now().Add(15*time.Second))
+	err := c.secureClientV2.SendPoliciesToAgents(ctx)
+	if err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Here we are adding code specific to policies. Sysdig clients is used for other parts of the sysdig, for monitor as well. What will happen if user authenticates only on monitor. secureClientV2 will be nil. Even if its not nil I would not like to have in this code anything releated to specific feature on our platform

Reply:
This file and SysdigClient is used to create clients depending on the product and environment not to perform actions on policies or any other feature.

Reply:
Modified per our discussion to create a generic concept of cleanup hooks that can be used by other features as well.